internal/cli/roles.go

```diff
@@ -29,6 +29,12 @@ var (
 		ShortForm: "d",
 		Help:      "Description of the role.",
 	}
+	roleNumber = Flag{
+		Name:      "Number",
+		LongForm:  "number",
+		ShortForm: "n",
+		Help:      "Number of roles, that match the search criteria, to retrieve. Maximum result number is 1000.",
```

Comment:
Default?

Reply:
Handled in d8c07f88597419f779b354ed158439265d41613a